types/errors/errors.go

```diff
@@ -318,12 +318,11 @@ func (e *wrappedError) Cause() error {
 
 // Is reports whether any error in e's chain matches a target.
 func (e *wrappedError) Is(target error) bool {
-	if target == nil {
-		return e == target
+	if e == target {
```

Comment:
Now what happens when target == nil?

Reply:
Now - you mean with this update? If `e` is also `nil` it will return true, otherwise it will recurse and check.

In `master` this test fails:

 	require.True(errw.Is(errw), "should match itself") 

But this passes:

 	require.True(stdlib.Is(errs, errs))